rllib/tuned_examples/ddpg/halfcheetah_2020-04-26/01_DDPG_HalfCheetah-v2_use_pytorch=True_2020-04-26/config.yaml

```diff
@@ -0,0 +1,39 @@
+actor_hiddens:
```

Comment:
Can we keep these yamls runnable with rllib train?

Reply:
All these have been removed again and placed into the github.com/ray-project/rl-experiments repo.